src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+const appendColum = document.querySelector('.append-column');
+const removeColum = document.querySelector('.remove-column');
```

Comment:
The variable name `removeColum` is misspelled. It should be `removeColumn` to reflect the correct spelling.